lang/lang-es.js

```diff
@@ -0,0 +1,594 @@
+    + ' <figure class="chrome" > <img src="https://www.congrea.com/wp-content/uploads/2016/10/ff-video.png" >'
+    + ' <figcaption>Chrome</figcaption> </figure></div>',
+
+    nopermissionExt: '<div class="errorMsg"> Se ha bloqueado el acceso a la cámara.'
+    + ' Para proporcionar acceso a la cámara web, siga los siguiente pasos <br />'
+    + 'Ir al ícono de la cámara en la esquina superior derecha de la pantalla <br /> '
+    + 'Haga clic en "Permitir siempre..." y seleccione la opción de cámara <br />  '
+    + 'Acepte y actualice la pantalla  </div> <div class="screenImages">'
+    + ' <figure class="chrome" > <img src="https://www.congrea.com/wp-content/uploads/2016/10/ff-video.png" >'
+    + ' <figcaption>Chrome</figcaption> </figure></div>',
+
+    chromeExtMiss: "Congrea necesita el plugin'Selector de escritorio' para compartir la pantalla.<br />"
+    + 'Puedes descargarlo desde <a '
+    + "href='https://chrome.google.com/webstore/detail/desktop-selector/ijhofagnokdeoghaohcekchijfeffbjl'"
+    + "target='_blank'>AQUÍ.</a>",
+
+
+    /* For Firefox */
+    PermissionDeniedErrorExtFF: '<div class="errorMsg"> Se ha bloqueado el acceso a la cámara.'
+    + ' Para proporcionar acceso a la cámara web, siga los siguiente pasos <br />'
+    + 'Ir al ícono de la cámara en la parte superior izquierda de la pantalla <br /> '
+    + 'Haga clic en "Compartir dispositivo seleccionado.".'
+    + '<div class="screenImages">'
+    + '<figure class="firefox" > <img src="https://www.congrea.com/wp-content/uploads/2016/10/ff-screnshare.png" >'
+    + ' <figcaption>Chrome</figcaption> </figure></div>',
+    screensharereload: 'La pantalla no se comparte. Vuelva a cargar para compartir.',
+
+    SecurityErrorExtFF: '<div class="errorMsg"> Se ha bloqueado el acceso a la cámara.'
+    + ' Para proporcionar acceso a la cámara web, siga los siguiente pasos <br />'
+    + 'Ir al ícono de la cámara en la parte superior izquierda de la pantalla <br /> '
+    + 'Haga clic en "Compartir dispositivo seleccionado.".'
+    + '<div class="screenImages">'
+    + ' <figure class="firefox" > <img src="https://www.congrea.com/wp-content/uploads/2016/10/ff-screnshare.png" >'
+    + ' <figcaption>Chrome</figcaption> </figure></div>',
+
+    nopermissionExtFF: '<div class="errorMsg"> Se ha bloqueado el acceso a la cámara.'
+    + ' Para proporcionar acceso a la cámara web, siga los siguiente pasos <br />'
+    + 'Ir al ícono de la cámara en la parte superior izquierda de la pantalla <br /> '
+    + 'Haga clic en "Compartir dispositivo seleccionado.".'
+    + '<div class="screenImages">'
+    + '<figure class="firefox" > <img src="https://www.congrea.com/wp-content/uploads/2016/10/ff-screnshare.png" >'
+    + '<figcaption>Chrome</figcaption> </figure></div>',
+    TypeError: 'Su webcam no está inicializada correctamente. Por favor recarga la página.',
+    newPage: 'Insertar página',
+  };
+  window.congreaLanguages.es = message;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assignment to property of function parameter 'window'.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)